cognite/neat/_rules/transformers/_converters.py

```diff
@@ -1703,6 +1808,36 @@ def convert_multi_data_type(value_type: MultiValueTypeInfo) -> DataType:
 
         return data_types.String()
 
+    def _get_cognite_concepts(self) -> set[ClassEntity]:
+        return {cls_.class_ for cls_ in self.rules.classes if str(cls_.class_.prefix) in COGNITE_SPACES} | {
+            parent
+            for cls_ in self.rules.classes
+            for parent in cls_.implements or []
+            if str(parent.prefix) in COGNITE_SPACES
+        }
+
+    @staticmethod
+    def _get_cognite_properties(concepts: set[ClassEntity], client: NeatClient) -> DMSRules:
+        view_ids = [dm.ViewId(str(cls_.prefix), cls_.suffix, cls_.version) for cls_ in concepts]
+        views = client.loaders.views.retrieve(view_ids, format="read", include_connected=True, include_ancestor=True)
+        spaces = Counter(view.space for view in views)
+        space = spaces.most_common(1)[0][0]
+        model: dm.DataModel[dm.View] = dm.DataModel(
```

Comment:
This DataModel you are instantiating here, is it only a placeholder right?
The parameters here seems arbitrary, so just to be sure I understood

Reply:
Yes, added some more description for it.